locales/en.json

```diff
@@ -461,5 +460,15 @@
   "Long-term care costs": "Long-term care costs",
   "Public or non-profit long-term care home": "Public or non-profit long-term care home",
   "Did you pay property tax?": "Did you pay property tax?",
-  "Property tax paid": "Property tax paid"
-}
+  "Property tax paid": "Property tax paid",
+  "start.p2.2": "under $12,000",
+  "start.p2.3": ". Income includes:",
+  "start.p3.1": "You ",
+  "start.p3.2": "cannot",
+  "start.p3.3": " use this service if you had income from your own business or self-employment.",
+  "start.p4": "You will need:",
+  "start.p5": "Your must answer extra security questions if you want your notice of assessment immediately after filing. You need information such as previous addresses and banking details to answer these questions.",
+  "start.h2-1": "Before you start",
+  "start.h2-2": "Help using the service",
+  "start.p6": "Contact CRA by:"
```

Comment:
Hey nick, can you group together all the ones that are `start.`? (ie, put them at the top)